cloudify_rollback_workflow/examples/rollback_to_configured_and_uninitialized.yaml

```diff
@@ -0,0 +1,59 @@
+tosca_definitions_version: cloudify_dsl_1_3
```

Comment:
blueprints should use dashes in the file name not underscores. It helps to distinguish them from python files (the other predominant language that we publish in.)